### PR TITLE
Workflow example

### LINKED ISSUE #412: Fake issue to demonstrate how the GH workflow can be used
The styling on this curriculum site needs an overhaul. Nothing feels loud enough. 

THIS IS NOT REAL -- PURELY FOR A MOD 2 LESSON

### PR DESCRIPTION
**Pull request by khalidwilliams**
#### What's this PR do?
This PR breaks a lot of styling. It's meant to be an example.
#### Where should the reviewer start?
Anywhere really, it's all bad. PLEASE DO NOT MERGE 

#### How should this be manually tested?
```bash
git fetch -a 
git checkout -b workflow-example 
bundle exec jekyll serve
```
#### Any background context you want to provide?
Again, just meant for review practice. Please don't merge.
#### What are the relevant tickets?
This closes #412 
#### Screenshots (if appropriate)
 Screen Shot 2019-10-17 at 6 46 29 PM

Screen Shot 2019-10-17 at 6 57 10 PM

#### Questions:
- Is there a blog post? N/A
- Does the knowledge base need an update? N/A
- Does this add new dependencies which need to be added? N/A

### CODE REVIEW

File: _sass/content.scss [hunk 1]
```diff
@@ -10,10 +10,11 @@
       margin-top: 1em;
     }
   }
+  background-color: blue;
```
Comment (benjamin-firth) [inline _sass/content.scss:13]: Love the color! - Ben

File: _sass/content.scss [hunk 2]
```diff
@@ -10,10 +10,11 @@
...
 }
 
 .content a:link, .content a:visited {
-  color: blue;
```
Comment (kaylagordon) [inline _sass/content.scss:16]: The blue makes the text hard to read. I would choose a lighter color here.

File: _sass/home-page.scss [hunk 3]
```diff
@@ -84,29 +84,6 @@
...
-    margin: 0px 0px 0px 50px;
-  }
-
-  .resources-section {
-    img {
-      margin: 10px;
-      width: 250px;
-    }
-  }
-}
```
Comment (Dustin-Mikusko) [inline _sass/home-page.scss:87]: This is great work Khalid. Tablets are overrated anyways

File: _sass/content.scss [hunk 4]
```diff
@@ -10,10 +10,11 @@
...
+  color: yellow;
```
Comment (cammac60) [inline _sass/content.scss]: I was thinking a more neon-green color here. This would really improve readability.

**Review by JamesRexMiller4 — COMMENTED** [review]
👻

File: lessons/module-2/github-collaboration.md [hunk 5]
```diff
@@ -7,21 +7,22 @@ module: 2
...
-
-* Give professional and actionable feedback through code reviews
-* Recognize what makes a README useful
-* File and organize issues with labels, that are closed through PRs
+__      __.__            __     ____ _____________   ____ _______________     ___________________________________________________
+/  \    /  \  |__ _____ _/  |_  |    |   \______   \ /_   /   __   \   _  \   /  __  \_____   \_____   \_____   \_____   \_____   \
+\   \/\/   /  |  \\__  \\   __\ |    |   /|     ___/  |   \____    /  /_\  \  >      <  /   __/  /   __/  /   __/  /   __/  /   __/
+\        /|   Y  \/ __ \|  |   |    |  / |    |      |   |  /    /\  \_/   \/   --   \|   |    |   |    |   |    |   |    |   |   
+ \__/\  / |___|  (____  /__|   |______/  |____|      |___| /____/  \_____  /\______  /|___|    |___|    |___|    |___|    |___|   
+      \/       \/     \/                                                 \/        \/ <___>    <___>    <___>    <___>    <___>
```
Comment (Alkiiro) [inline lessons/module-2/github-collaboration.md:16]: This isn't code.  How dare you.

File: _sass/variables.scss [hunk 6]
```diff
@@ -1,12 +1,12 @@
-$turing-primary: #12F1FC;
+$turing-primary: #83831f;
 $turing-secondary: #036575;
-$front-end-primary: #F9AE06;
+$front-end-primary: #83831f;
 $front-end-secondary: #965D0B;
-$back-end-primary: #EF3852;
+$back-end-primary: #83831f;
```
Comment (adamsjr8576) [inline _sass/variables.scss:5]: Definitely makes it pop more!

File: _sass/content.scss (hunk 2 above)
Comment (Dustin-Mikusko) [reply _sass/content.scss:16]: I respectfully disagree. Most people's retinas inherently receive blue very well

Comment (trishalanglois) [reply _sass/content.scss]: I actually really like this blue.  @kaylaewood, maybe you should get your eyes checked.  Let me know if you need a good ophthalmologist.

File: _sass/content.scss (hunk 1 above)
Comment (jpweist) [reply _sass/content.scss:13]: but we want green

File: _sass/navbar.scss [hunk 7]
```diff
@@ -1,86 +0,0 @@
...
-      grid-column: 1;
-      grid-row: 1 / 5;
-    }
-
-    .home-logo-link img {
-      height: 70px;
-      width: 70px;
-    }
-  }
-}
```
Comment (hfaerber) [inline _sass/navbar.scss:86]: @khalidwilliams  We may want to consider responsiveness for different screen sizes and leave these media queries.

File: _sass/content.scss (hunk 4 above)
Comment (khalidwilliams) [reply _sass/content.scss]: Love the input! I think that could work, will push up soon

File: lessons/module-2/github-collaboration.md [hunk 8]
```diff
@@ -7,21 +7,22 @@ module: 2
 
 ## Goals
 
-By the end of this lesson, students should be able to:
-
-* Give professional and actionable feedback through code reviews
-* Recognize what makes a README useful
-* File and organize issues with labels, that are closed through PRs
+__      __.__            __     ____ _____________   ____ _______________     ___________________________________________________
```
Comment (spaceplesiosaur) [inline lessons/module-2/github-collaboration.md:10]: This is fun, but I don't think it should replace the goals of the lesson, as students find those useful.

File: lessons/module-2/github-collaboration.md [hunk 9]
```diff
@@ -7,21 +7,22 @@ module: 2
...
+/  \    /  \  |__ _____ _/  |_  |    |   \______   \ /_   /   __   \   _  \   /  __  \_____   \_____   \_____   \_____   \_____   \
+\   \/\/   /  |  \\__  \\   __\ |    |   /|     ___/  |   \____    /  /_\  \  >      <  /   __/  /   __/  /   __/  /   __/  /   __/
+\        /|   Y  \/ __ \|  |   |    |  / |    |      |   |  /    /\  \_/   \/   --   \|   |    |   |    |   |    |   |    |   |   
+ \__/\  / |___|  (____  /__|   |______/  |____|      |___| /____/  \_____  /\______  /|___|    |___|    |___|    |___|    |___|   
+      \/       \/     \/                                                 \/        \/ <___>    <___>    <___>    <___>    <___>
```
Comment (sertmer) [inline lessons/module-2/github-collaboration.md:15]: SIIIIIICK

File: lessons/module-2/github-collaboration.md (hunk 8 above)
Comment (benjamin-firth) [inline lessons/module-2/github-collaboration.md:10]: Please keep this!

File: _sass/home-page.scss [hunk 10]
```diff
@@ -84,29 +84,6 @@
...
 
 @media (max-width: $phone-width) {
   .splash {
```
Comment (MattTuring) [inline _sass/home-page.scss:89]: This doesnt work very well on my samsung smart fridge

File: lessons/module-2/github-collaboration.md (hunk 5 above)
Comment (hfaerber) [inline lessons/module-2/github-collaboration.md:16]: Very Cool.  The turing site really needed this direct shout out to 1908 in my opinion.

File: _sass/navbar.scss [hunk 11]
```diff
@@ -1,86 +0,0 @@
-nav {
```
Comment (andrewckinstler) [inline _sass/navbar.scss:1]: Great work removing unnecessary code!

File: _sass/content.scss (hunk 4 above)
Comment (RayRedGoose) [reply _sass/content.scss]: I totally agree with you! Now this color is very boring

Comment (JamesRexMiller4) [reply _sass/content.scss]: 🦔

File: _sass/fonts.scss [hunk 12]
```diff
@@ -5,32 +5,32 @@ $light-copy: 'Helvetica Neue Light';
...
+//   src: url('../assets/fonts/HelveticaNeueLTStd-Hv.otf');
+// }
+//
+// @font-face {
+//   font-family: 'Helvetica Neue Bold';
+//   src: url('../assets/fonts/HelveticaNeueLTStd-Bd.otf');
+// }
+//
+// @font-face {
+//   font-family: 'Helvetica Neue Standard';
```
Comment (Dunn-Austen) [inline _sass/fonts.scss]: How do you pronounce Neue?

File: _sass/content.scss (hunk 4 above)
Comment (khalidwilliams) [reply _sass/content.scss]: @cammac60 updated the color in 5da1142, check it out!

Comment (Alkiiro) [reply _sass/content.scss]: I've never seen JimJames' GH profile pic before....what're you doing, posing for a red-carpet photo shoot?!

Comment (andrewckinstler) [reply _sass/content.scss]: @Alkiiro channeling his inner McConaughey

**Pull request closed without merging**